extensions/wikia/Discussions/Discussions.setup.php

```diff
@@ -13,14 +13,18 @@
 	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/SpecialDiscussions',
 ];
 
-// classes
-$wgAutoloadClasses['SpecialDiscussionsController'] = $dir . 'SpecialDiscussionsController.class.php';
+// load classes
+$wgAutoloadClasses['SpecialDiscussionsController'] = $dir . 'controllers/SpecialDiscussionsController.class.php';
+$wgAutoloadClasses['EnableDiscussionsController'] = $dir . 'controllers/EnableDiscussionsController.class.php';
+$wgAutoloadClasses['DiscussionsActivator'] = $dir . 'utils/DiscussionsActivator.class.php';
+$wgAutoloadClasses['DiscussionsVarToggler'] = $dir . 'utils/DiscussionsVarToggler.class.php';
+$wgAutoloadClasses['DiscussionsVarTogglerException'] = $dir . 'utils/DiscussionsVarToggler.class.php';
 
-// special page
+// register special page
 $wgSpecialPages['Discussions'] = 'SpecialDiscussionsController';
 
 // message files
-$wgExtensionMessagesFiles['SpecialDiscussions'] = $dir . 'SpecialDiscussions.i18n.php';
+$wgExtensionMessagesFiles['SpecialDiscussions'] = $dir . 'i18n/SpecialDiscussions.i18n.php';
 
 // permissions
 $wgAvailableRights[] = 'specialdiscussions';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "specialdiscussions" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)